src/Database/Redis/PubSub.hs

```diff
@@ -83,6 +101,14 @@ sendCmd cmd       = do
     lift $ Core.send (redisCmd cmd : changes cmd)
     modifyPending (updatePending cmd)
 
+totalPendingChanges :: PubSub -> Int
+totalPendingChanges (PubSub{..}) =
+  updatePending subs $ updatePending unsubs $ updatePending psubs $ updatePending punsubs 0
```

Comment:
and that also makes me think that `updatePending` is a bit confusing name here as there no actual "update" in it something like `addPending` seems to look better